Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.12.2-alpine3.19
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
```

Comment:
The `WORKDIR` path should not end with a slash. Please change it to `WORKDIR /app` to ensure consistency and avoid potential issues.